pkg/option/config.go

```diff
@@ -2429,6 +2471,10 @@ func (c *DaemonConfig) Populate() {
 
 	c.populateDevices()
 
+	c.IPAMClusterPoolV4CIDR = viper.GetStringSlice(IPAMClusterPoolV4CIDR)
+	c.validateIPAMClusterPoolV4CIDR()
+	c.IPAMClusterPoolV6CIDR = viper.GetStringSlice(IPAMClusterPoolV6CIDR)
+	c.validateIPAMClusterPoolV6CIDR()
```

Comment:
These lines are ignoring the returned error code.

Reply:
Addressed review comments.